src/background/sidebar-injector.ts

```diff
@@ -347,7 +347,7 @@ export class SidebarInjector {
       }
 
       const onMessage = chromeAPI.runtime.onMessage;
-      const listener = (request, sender, sendResponse) => {
+      const listener = (request: any, sender: any, sendResponse: any) => {
```

Comment:
The Chrome extensions types that we use might already have a more specific type that is appropriate for `listener` here. If not, this will be OK.

Reply:
Good point. I will check that.